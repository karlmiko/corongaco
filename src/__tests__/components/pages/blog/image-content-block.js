import React from 'react'
import renderer from 'react-test-renderer'
import ImageContentBlock from '~components/pages/blog/image-content-block'

describe('Components : Pages : Blog : Image Content Block', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <ImageContentBlock
          image={{
            'en-US': {
              fields: {
                file: {
                  'en-US': {
                    url: '//images.ctfassets.net/anexampleimage',
                  },
                  title: {
                    'en-US': 'This is a test image title',
                  },
                },
                title: 'An Example Image',
              },
            },
          }}
          caption={{
            'en-US': 'This is an image caption',
          }}
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
