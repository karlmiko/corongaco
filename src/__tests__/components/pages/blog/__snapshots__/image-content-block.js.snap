// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Blog : Image Content Block renders correctly 1`] = `
<div>
  <img
    src="//images.ctfassets.net/anexampleimage"
  />
  <div
    className="credit"
  >
    This is an image caption
  </div>
</div>
`;
